frontend/src/pages/org/crawls-list.ts

```diff
@@ -884,8 +817,16 @@ export class CrawlsList extends LiteElement {
       .split("/orgs/")[1]
       .split("/")[0];
     try {
-      const { names, firstSeeds, workflowIds } = await this.apiFetch(
-        `/orgs/${oid}/crawlconfigs/search-values`,
+      const query = queryString.stringify({
+        crawlType: this.artifactType,
+      });
+      const data: {
+        crawlIds: string[];
+        names: string[];
+        descriptions: string[];
+        firstSeeds: string[];
+      } = await this.apiFetch(
+        `/orgs/${oid}/all-crawls/search-values?${query}`,
```

Comment:
@tw4l I proactively added the filter query for `search-values`, if you want to work off of this branch to test it out.